charts/README.md

```diff
@@ -0,0 +1,88 @@
+
+- [sda-svc - SDA services](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-svc/README.md)
+
+  This chart deploys the service components needed to operate the Sensitive Data Archive solution. The charts may include additional service components that might be beneficial for administrative operations or extending the Sensitive Data Archive solutions to facilitate different use cases.
+
+- [sda-db - SDA database](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-db/README.md)
+
+  This chart deploys a pre-configured database ([PostgreSQL](https://www.postgresql.org/)) instance for Sensitive Data Archive, the database schemas are designed to adhere to [European Genome-Phenome Archive](https://ega-archive.org/) federated archiving model.
+
+- [sda-mq - SDA Message broker](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-mq/README.md)
+
+  This chart deploys a pre-configured message broker ([RabbitMQ](https://www.rabbitmq.com/)) designed for [European Genome-Phenome Archive](https://ega-archive.org/) federated messaging between `CentralEGA` and Local/Federated EGAs but also configurable to support Standalone SDA deployments.
+
+- [sda-orch - SDA orchestrate service](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-orch/README.md)
+
+  This chart deploys an orchestration service for the Sensitive Data Archive solution. This is a helper service designed to curate the ingestion flow in an automated manner when the SDA solution is deployed and configured as standalone (non-federated).
+
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+With Helm properly installed, add the `neicnordic` Helm repository as follows:
+
+```sh
+helm repo add neicnordic https://neicnordic.github.io/sensitive-data-archive
+helm repo update
+```
+
+You can then run
+
+```sh
+helm search repo neicnordic
+```
+
+to see the available charts.
+
+## Installing the Charts
+
+To install a chart with the release name `my-release`:
+
+```sh
+helm install my-release neicnordic/<chart-name>
+```
+
+To configure a Helm chart with your own values, you can copy the default `values.yaml` file from the chart to your local directory and modify it as needed, or using helm:
+
+```sh
+helm show values neicnordic/<chart-name> > <values-filename>.yaml
+```
+
+Then, you can install the chart with the following command:
+
+```sh
+helm install my-release -f <values-filename>.yaml neicnordic/<chart-name>
+```
+
+Example:
+
+```sh
+helm show values neicnordic/sda-svc > my-values.yaml
+vi my-values.yaml
+helm install my-release neicnordic/sda-svc -f my-values.yaml
+```
+
+For quick reference to Helm's chart management capabilities see [here](https://helm.sh/docs/intro/cheatsheet/#chart-management).
```

Comment:
This section should mention that the secrets containing the crypt4gh key pair and passphrase needs to be crated by the admin before the chart is deployed.
The same goes for the JWT key if resigning is enabled.

Reply:
Good point, I added the necessary commands so that the example is more illustrative.